### ISSUE TITLE
Reproducing 2d results

### ISSUE THREAD

**Issue by Atcold** [taldatech/soft-intro-vae-pytorch#17]
What are the 12 command line instructions necessary to reproduce these results?

![image](https://user-images.githubusercontent.com/2119355/223535912-2fcb16f1-e577-4939-9e26-7ba33d8ed7eb.png)

**Comment by taldatech** [comment]
Hi,
We used the hyper-parameters that can be found in the appendix and the notebook here:
https://github.com/taldatech/soft-intro-vae-pytorch/blob/main/soft_intro_vae_tutorial/soft_intro_vae_2d_code_tutorial.ipynb

device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
print("device:", device)
seed = 92  # for reproducible results
datasets = ['8Gaussians', '2spirals', 'checkerboard', 'rings']
chosen_hyperparmas = {'8Gaussians': {'b_kl': 0.3, 'b_neg': 0.9, 'b_rec': 0.2},
                      '2spirals': {'b_kl': 0.5, 'b_neg': 1.0, 'b_rec': 0.2},
                      'checkerboard': {'b_kl': 0.1, 'b_neg': 0.2, 'b_rec': 0.2},
                      'rings': {'b_kl': 0.2, 'b_neg': 1.0, 'b_rec': 0.2}}

num_iter = 30_000
lr = 2e-4
batch_size = 512

![image](https://user-images.githubusercontent.com/30633883/223610999-1a7ec27e-691d-4279-82ed-06f93ab20e9b.png)

Also, in: https://github.com/taldatech/soft-intro-vae-pytorch/blob/main/soft_intro_vae_2d/train_soft_intro_vae_2d.py

 """
        Recommended hyper-parameters:
        - 8Gaussians: beta_kl: 0.3, beta_rec: 0.2, beta_neg: 0.9, z_dim: 2, batch_size: 512
        - 2spirals: beta_kl: 0.5, beta_rec: 0.2, beta_neg: 1.0, z_dim: 2, batch_size: 512
        - checkerboard: beta_kl: 0.1, beta_rec: 0.2, beta_neg: 0.2, z_dim: 2, batch_size: 512
        - rings: beta_kl: 0.2, beta_rec: 0.2, beta_neg: 1.0, z_dim: 2, batch_size: 512
    """

I hope this helps

**Comment by Atcold** [comment]
Thanks!
